src/System.Security.Cryptography.X509Certificates/src/System/Security/Cryptography/X509Certificates/X509Certificate2Collection.cs

```diff
@@ -105,9 +97,6 @@ public void AddRange(X509Certificate2Collection certificates)
 
         public bool Contains(X509Certificate2 certificate)
         {
-            if (certificate == null)
```

Comment:
Since you added a test, someone would have to go really out of their way to put it back; so probably not worth messing with the commit integrity.